lib/main_common.pm

```diff
@@ -1155,8 +1154,6 @@ sub load_x11tests {
     loadtest "x11/xterm";
     loadtest "x11/sshxterm" unless get_var("LIVETEST");
     if (gnomestep_is_applicable()) {
-        # openSUSE has an explicit update check elsewhere
-        loadtest "update/updates_packagekit_gpk" if is_sle && !is_staging;
```

Comment:
yes, it's good if you want to remove it from here but I think you are doing it the wrong way around. Do not define a SLE specific function elsewhere but find out why SLE calls it here and openSUSE does not. And then try to find a way which is common for both openSUSE and SLE